### PR TITLE
Add store book

### PR DESCRIPTION
**Pull request by lubnaabd**
## Description

create addNewBook front-end and back-end code

### Type of change

New features #64
Description : create book_store.js that contain fetch function that make a request (post) to back-end and contain validation function.

New features #65 
Description : 
* add  function that deal with the request from the front-end in book.js in the controller 

New features #16
Description :
- create setStore query, that add all input in db.

New features testing
- test route addBookStore.
- test setStore query.

### CODE REVIEW

File: public/js/book_store.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      body: JSON.stringify(data),
+    })
+      .then(result => result.json())
+      .then((response) => {
+        if (response.errorMessage) return swal('Error !', response.errorMessage, 'error');
+        swal('Good job!', response.message, 'success').then((value) => {
+          JSON.stringify(response);
+        });
+      })
+      .catch(error => swal('Error while adding category !', error.errorMessage, 'error'));
```
Comment (RamyAlshurafa) [inline public/js/book_store.js]: `Error while adding category` while you are adding books!!

File: src/controllers/test/test.js [hunk 2]
```diff
@@ -286,15 +285,48 @@ test('test for add library view page route - with cookie and auth ', (t) => {
...
+      bookId: 4,
+      copyNumberVal: 10,
+    })
+    .expect(200)
+    .expect('Content-Type', /html/)
+    .end((err, res) => {
+      if (err) {
+        t.error(err);
+      }
+      t.equal(res.res.statusMessage, 'OK', 'statusMessage should return OK');
```
Comment (RamyAlshurafa) [inline src/controllers/test/test.js]: `statusMessage` for `200` is `'OK'` always so this is same as line 321
I think it's better to check `JSON.parse(res.text).message` is it the same message you sent?

File: public/css/book.css [hunk 3]
```diff
@@ -136,6 +138,13 @@
   padding: 0.3rem;
   float: left;
 }
+.button--view {
```
Comment (RamyAlshurafa) [inline public/css/book.css:142]: add a white line here

**Review by RamyAlshurafa — CHANGES_REQUESTED** [review]
small notes

**Review by RamyAlshurafa — APPROVED** [review]
#### Approved :ok_hand:

**Review by ali-7 — APPROVED** [review]

**Review by RamyAlshurafa — APPROVED** [review]

**Pull request merged**